recipe/meta.yaml

```diff
@@ -1,15 +1,24 @@
 {% set version = "2.5.1" %}
+{% set sha256 = "740eb5fff95e33cfe699bad43be83523f569c7cc7f9c285c2a255416443dd266" %}
+# Set the RC number to build release candidates. Set to None otherwise
+{% set rc = None %}
 {% set build = 1 %}
```

Comment:
Should we reset the build number?

Reply:
this was sync'd with conda-forge at build number 1. Usually we try and keep in line with their build numbers for ease of sync, but it's a pretty minor tradeoff point.